src/content/docs/validator/external-chains/stellar.mdx

```diff
@@ -844,19 +844,26 @@ In your `config.toml` file for ampd you can set the following configuration to r
     </tab-item>
     <tab-item title="Testnet" class="bleed">
         ```toml
-        [[handlers]]
-        type="StellarMsgVerifier"
-        cosmwasm_contract="axelar1j82crkl36gezwlch3ngkcyva9gjrqsz4vr76u3uvazdl7z70sfdsx30ma2"
-        rpc_url="your_own_rpc"
-
-        [[handlers]]
-        type="StellarVerifierSetVerifier"
-        cosmwasm_contract="axelar1j82crkl36gezwlch3ngkcyva9gjrqsz4vr76u3uvazdl7z70sfdsx30ma2"
-        rpc_url="your_own_rpc"
+      [[handlers]]
+      type="StellarMsgVerifier"
+      cosmwasm_contract="axelar1a4wt84rllhuwpdvymj4tql6cugfsmdmau9ufmzcm329yx9st96eqx05uam"
+      rpc_url="your_own_prc"
+
+      [[handlers]]
+      type="StellarVerifierSetVerifier"
+      cosmwasm_contract="axelar1a4wt84rllhuwpdvymj4tql6cugfsmdmau9ufmzcm329yx9st96eqx05uam"
+      rpc_url="your_own_prc"
         ```
     </tab-item>
 </tabs>
 
+### Register Chain 
+For your node to be recognized by the Axelar network, you need to register your chain with the `ampd` daemon. This can be done by running:  
+
+```bash
+ampd register-chain-support amplifier stellar
```

Comment:
can we also make more prominent that they will need to have their ed25519 keys registered, and in order for that to work, they need to be on tofnd 1.0.1?